### DISCUSSION TITLE
Interested in how to connect github discussions in deeper

### DISCUSSION THREAD

**Discussion by techsmyth** [alkem-io/alkemio#347] [Ideas] [answered]
how to optimally integrate the capabilities in discussions with the alkemio hub?

**Comment by FloriSpoelstra** [comment, marked as answer]
I think it would be nice if there is at least a link on the hub to this forum. It should at minimum be easily accessible for admins, but maybe even for everyone.

Same for the development backlog/roadmap. It would add to the feel of a community platform.

**Reply by techsmyth** [reply]
There is a link to the support page in the footer, but it may need to become more prominent:
![image](https://user-images.githubusercontent.com/30729240/128331502-f8bb782c-818a-4a14-9449-c6152b08ed37.png)

The support page in turn directs to discussions...

Interesting point regarding the development backlog / roadmap. How would you see that displayed? It may be that we want to put a more prominent display of the link to the github repo + from there direct?

**Reply by FloriSpoelstra** [reply]
Yes I would make this way more prominent, maybe by even adding them to the menu bar:

![Screenshot 2021-08-05 12 18 06](https://user-images.githubusercontent.com/87815729/128334584-f39a880b-5972-43a5-891e-95934fff06ec.png)
